src/lib/falcor/routes/articles/__tests__/by-page.test.js

```diff
@@ -0,0 +1,102 @@
+import { articlesByPageRoutes } from '../by-page';
+
+jest.enableAutomock();
+jest.unmock('../by-page');
+jest.unmock('lodash');
+jest.unmock('falcor');
+
+describe('articlesByPage falcor routes', () => {
+  // More to make sure we didn't miss any
+  it('has only 1 route', () => {
+    expect(articlesByPageRoutes.length).toBe(1);
+  });
+
+  it('is the right route we are testing', () => {
+    expect(articlesByPageRoutes[0].route).toBe(
```

Comment:
This would break in case someone updates the articlesByPageRoutes and adds another route on top of the previous one. I would rather do `articlesByPageRoutes.findIndex(...)`, to get the proper index, and then test on that index for future extensibility.

Reply:
Nevermind this comment in case you intend that the routes files can only have at most one route in general. I approved it.

Reply:
No, normally most of them would have 2-4 routes, so I actually quite like your solution, I was thinking of how to do it well since it's an array. I'll try and change the implementation to follow your idea :)

Reply:
Alright, let me know once it is updated, and I'll approve the PR :)